app.go

```diff
@@ -404,16 +417,16 @@ func HandleAction(action interface{}, context *Context) error {
 	if len(vals) == 0 {
 		fmt.Fprintln(os.Stderr,
 			"DEPRECATED Action signature.  Must be `cli.ActionFunc`")
-		return nil
+		return err
```

Comment:
Wait, maybe, now I've confused myself. Is there a case where this wouldn't return `nil`?